scripts/changelog.sh

```diff
@@ -21,12 +21,18 @@ Commands:
     type. The release type should be one of "dev", "alpha", "rc", "release", or "patch".
     `dev`: will update the changelog with the latest unreleased changes.
     `alpha`: will generate a new section with an alpha version for today.
+    `beta`: will generate a new beta release.
     `rc`: will generate a new rc release.
     `release`: will make the initial minor release for this branch.
     `patch`: will generate a new patch release
 
   nextminor
-    Run this to get a new release branch for the next minor version.
+    This function expects the current branch to be main. Run it if you want to set main to the next
+    minor version.
+    
+  maintenancerelease
```

Comment:
Could you add a short description of what this new command does?

Reply:
Yeah sure, forgot to copy it over